shell/renderer/extensions/electron_extensions_renderer_client.cc

```diff
@@ -16,12 +16,6 @@ namespace electron {
 
 ElectronExtensionsRendererClient::ElectronExtensionsRendererClient() {}
 
-void ElectronExtensionsRendererClient::RenderThreadStarted() {
-  if (!dispatcher()) {
-    CreateDispatcher();
-  }
```

Comment:
I don't understand why this isn't needed anymore?

Reply:
@ckerr https://chromium-review.googlesource.com/c/chromium/src/+/5786668/3